src/main/common.c

```diff
@@ -575,7 +531,22 @@ key_append(const as_val* key, const as_val* val, void* udata)
 LOCAL_HELPER int
 key_cmp(const void* v1, const void* v2)
 {
-	return as_val_cmp(*(as_val**)v1, *(as_val**)v2);
+	// why offset by 1 here because as_val_cmp considers 0 as less than, 1 as equal, and 2 as greater.
```

Comment:
"why offset by 1 . . ." Maybe "We offset by 1"

Reply:
changed to "we ..."